builtin/providers/vsphere/provider.go

```diff
@@ -29,6 +29,13 @@ func Provider() terraform.ResourceProvider {
 				DefaultFunc: schema.EnvDefaultFunc("VSPHERE_VCENTER", nil),
 				Description: "The vCenter Server name for vSphere API operations.",
 			},
+
+			"insecure": &schema.Schema{
```

Comment:
I think we should consider naming this `allow_unverified_ssl` such that it consistent with the [Rundeck provider](https://github.com/hashicorp/terraform/blob/master/builtin/providers/rundeck/provider.go#L25-L30), and in order that it describes more clearly exactly which form of insecurity is invited?